types/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "werkit",
+  "version": "0.12.0",
```

Comment:
Have you thought about how you will keep the version of the node package in sync with the python package?

Reply:
I'm expecting it to be a manual part of the version-bumping process. If you think it's useful I could add a test that checks they are in sync.

Reply:
Yes, maybe as npm prepublishOnly script.

Reply:
Updated! bcef51a